pyopenms_viz/_matplotlib/__init__.py

```diff
@@ -35,7 +35,10 @@ def plot(data, kind, **kwargs):
     if plot_obj.show_plot:
         return plot_obj.show()
```

Comment:
Based on the feedback from the paper, do we want to return the actual figure object, even if `show_plot` is set to true?

Reply:
Yes I will change this. I think I changed this originally because docs look strange when an object is returned but I can instead just change the behaviour if it is a sphinx build